packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -25,6 +27,8 @@ class PolymarketNotifier {
     this.apiEndpoint = apiEndpoint;
     this.maxTimeAfterProposal = maxTimeAfterProposal;
     this.minAcceptedPrice = minAcceptedPrice;
+    // Manually add polymarket abi to the abi decoder global so aggregateTransactionsAndCall will return the correctly decoded data.
+    TransactionDataDecoder.getInstance().abiDecoder.addABI(abi);
```

Comment:
shouldn't abi Decoder automatically be able to see all contract ABI's or is this a special case?

Reply:
This is a special case because this abi is manually included in json form. This contract doesn't exist in core.